tests/src/test/java/io/streamnative/pulsar/handlers/kop/BasicEndToEndTestBase.java

```diff
@@ -331,4 +334,39 @@ protected void kafkaConsumeCommitMessage(KConsumer kConsumer,
         }
     }
 
+
+    @Test(timeOut = 20000)
+    public void testDeletePartition() throws Exception {
+        final String topic = "test-delete-partition";
+
+        pulsar.getAdminClient().topics().createPartitionedTopic(topic, 2);
+
+        @Cleanup
+        final KafkaProducer<String, String> kafkaProducer = newKafkaProducer();
+        sendSingleMessages(kafkaProducer, topic, Arrays.asList("a", "b", "c"));
+
+        List<String> expectValues = Arrays.asList("a", "b", "c");
+
+        @Cleanup
+        final KafkaConsumer<String, String> kafkaConsumer = newKafkaConsumer(topic);
+        List<String> kafkaReceives = receiveMessages(kafkaConsumer, expectValues.size());
+        Assert.assertEquals(kafkaReceives.stream().sorted().collect(Collectors.toList()), expectValues);
+
+        sendSingleMessages(kafkaProducer, topic, Arrays.asList("d", "e", "f"));
+        expectValues = Arrays.asList("d", "e", "f");
+        kafkaReceives = receiveMessages(kafkaConsumer, expectValues.size());
+        Assert.assertEquals(kafkaReceives.stream().sorted().collect(Collectors.toList()), expectValues);
+
+
+        sendSingleMessages(kafkaProducer, topic, Arrays.asList("g", "h", "i"));
+
+
+        pulsar.getAdminClient().topics().delete(topic + "-partition-1", true);
+        // "h" is usually written to Partition 1, so deleting partition-1 means that we lose "h"
+        expectValues = Arrays.asList("g", "i");
+
+        kafkaReceives = receiveMessages(kafkaConsumer, expectValues.size());
+        Assert.assertEquals(kafkaReceives.stream().sorted().collect(Collectors.toList()), expectValues);
```

Comment:
Use `assertEquals` instead of `Assert.assertEquals` to fix Codacy check

Reply:
thanks. fixed